sources/packages/api/src/route-controllers/student/student.controller.ts

```diff
@@ -636,23 +636,30 @@ export class StudentController extends BaseController {
    * by student uploader
    * @returns list of student documents
    */
-  @AllowAuthorizedParty(AuthorizedParties.student)
+  @AllowAuthorizedParty(AuthorizedParties.student, AuthorizedParties.aest)
```

Comment:
mm, I believe we discussed not to use multiple `AllowAuthorizedParty` to a single controller right as part of the new POC and here  Dto is also different for both controllers. with the new POC, we discussed we will create a separate controller for aest  something like `student.aest.controller.ts`